app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -160,7 +161,14 @@ const getChartOptions = (widget, options) => {
           if (!value) {
             return '';
           }
-          const totalValue = totalDataset.data[tooltipItem.index];
+          const isDefectType = /defects/.test(dataset.label);
+          const totalDefectsValue = defects.reduce(
```

Comment:
I think it superfluous to calculate statistics on defects for any label every time.
Maybe it's better to transfer this calculation to the condition block?
F.e.

```
          let totalValue;
          const isDefectType = /defects/.test(dataset.label);

          if (isDefectType) {
            const defects = data.datasets.filter((item) => /defects/.test(item.label));

            totalValue = defects.reduce(
              (total, defect) => total + defect.data[tooltipItem.index],
              0,
            );
          } else {
            totalValue = totalDataset.data[tooltipItem.index];
          }
```